SpatialGDK/Documentation/content/dynamic-typebindings.md

```diff
@@ -4,10 +4,10 @@ To allow Unreal to replicate through the SpatialOS network stack and combine mul
 
 `Dynamic Typebindings` operate at runtime so that your development iteration speed is not affected, despite your network code running on a completely different representation to Unreal’s.
 
-At the heart of `Dynamic Typebindings` is GDK-generated [SpatialOS schema]({{urlRoot}}/content/spatialos-concepts/schema), which is the SpatialOS representation of any Unreal object, its replicated data, and RPCs. `Dynamic Typebindings` also include the binding code that is invoked when the GDK converts network-relevant data between native Unreal and SpatialOS.
+At the heart of `Dynamic Typebindings` is GDK-generated [SpatialOS schema]({{urlRoot}}/content/spatialos-concepts/schema-and-snapshots#schema), which is the SpatialOS representation of any Unreal object, its replicated data, and RPCs. `Dynamic Typebindings` also include the binding code that is invoked when the GDK converts network-relevant data between native Unreal and SpatialOS.
 
 # Schema
-You generate the [schema]({{urlRoot}}/content/spatialos-concepts/schema) used in `Dynamic Typebindings` via the *Schema** button in the GDK toolbar. When you select **Schema**, the GDK generates schema for all classes tagged with the [Spatial Type]({{urlRoot}}/content/spatial-type) specifier. For each Unreal object that the GDK generates schema for, there are a number of possible schema components generated, each serving a different function.
+You generate the [schema]({{urlRoot}}/content/spatialos-concepts/schema-and-snapshots#schema) used in `Dynamic Typebindings` via the *Schema** button in the GDK toolbar. When you select **Schema**, the GDK generates schema for all classes tagged with the [Spatial Type]({{urlRoot}}/content/spatial-type) specifier. For each Unreal object that the GDK generates schema for, there are a number of possible schema components generated, each serving a different function.
```

Comment:
```suggestion
You generate the [schema]({{urlRoot}}/content/spatialos-concepts/schema-and-snapshots#schema) used in `Dynamic Typebindings` via the *Schema** button in the GDK toolbar. When you select **Schema**, the GDK generates schema for classes tagged with the [Spatial Type]({{urlRoot}}/content/spatial-type) specifier. For each Unreal object that the GDK generates schema for, there are a number of possible schema components generated, each serving a different function.
```
Suggest remove "all" as it's either a full generation or an incremental one, depending on the option you choose (as it's now changed). But we don't need to go into deets.